src/wp-admin/includes/class-custom-image-header.php

```diff
@@ -50,7 +50,7 @@ class Custom_Image_Header {
 	 * Constructor - Register administration header callback.
 	 *
 	 * @since 2.1.0
-	 * @param callable $admin_header_callback
+	 * @param callable $admin_header_callback. Default false.
```

Comment:
In the PR description and Trac Ticket you mentioned `$admin_image_div_callback` param to be updated, but the actual change is related to the `$admin_header_callback`

```suggestion
	 * @param callable $admin_header_callback.
```

Reply:
Thank you @cadic 
I've made changes and sent another PR. 

Could you please check and let me know how that goes?